### ISSUE TITLE
[CoE Starter Kit - BUG] PBI App Usage Details - for things like who launches the app 

### ISSUE THREAD

**Issue by nasilvae** [microsoft/coe-starter-kit#8348]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Team, 

Can we update the "App Launches per App Maker" to "App Launches per User" since this is usage is coming from all the usage of apps in the platform regardless if they built the app or not. 

![image](https://github.com/microsoft/coe-starter-kit/assets/95443860/992c92c3-2496-4568-a513-15ea8dc5a3da)

### Expected Behavior

Refine details. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.41

### What app or flow are you having the issue with?

Power BI Report

### What method are you using to get inventory and telemetry?

Cloud flows

AB#3357

**Comment by nasilvae** [comment]
I noticed we are using app owners as a way to filter users. Is there a way we can use the users from the audit table to get everything?

**Comment by Jenefer-Monroe** [comment]
Sorry I didnt make these so I'm just trying to sort them out in my head myself. 

This visual is actually about the virality of people's work, not about how often they launch apps. 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/64de401d-dec6-4009-87eb-a959df05c3b3)

So in this visual its not that Lynne launches a bunch of apps, its that their apps are launched a lot.

**Comment by Jenefer-Monroe** [comment]
If we were to add another visual I think it might need to be on a separate screen just due to real estate and purpose. 
Can put that on the backlog for consideration. Something like a screen that tracks where apps are used?

**Comment by nasilvae** [comment]
> If we were to add another visual I think it might need to be on a separate screen just due to real estate and purpose. 
> 
> Can put that on the backlog for consideration. Something like a screen that tracks where apps are used? 

yea along those lines of tracking usage of apps and if these apps are premium. Im going to work on a custom report on it will let you know how it goes.

**Comment by mohamrizwa** [comment]
Hi @nasilvae, 
We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the current Issue. 
Thank you for your understanding and cooperation.

**Comment by nasilvae** [comment]
> Hi [@nasilvae](https://github.com/nasilvae), We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the current Issue. Thank you for your understanding and cooperation.

Do we have confirmation the issue has been addressed? I have not seen any update from your team regarding the issue. @mohamrizwa . Please reopen the issue as we have not gotten confirmation of being addressed.

**Comment by mohamrizwa** [comment]
Hi @nasilvae,
Apologies for this.
Please proceed with upgrading to the latest version of the Center of Excellence (CoE) Starter Kit. Before starting, ensure all unmanaged layers are identified and removed to avoid deployment issues. 
For detailed guidance, refer to the below -mentioned 
 official documentation: 

 https://learn.microsoft.com/power-platform/guidance/coe/after-setup 

Feel free to reach out if you need any assistance. We're happy to help!

**Comment by nasilvae** [comment]
> Hi [@nasilvae](https://github.com/nasilvae), Apologies for this. Please proceed with upgrading to the latest version of the Center of Excellence (CoE) Starter Kit. Before starting, ensure all unmanaged layers are identified and removed to avoid deployment issues. For detailed guidance, refer to the below -mentioned official documentation:
> 
> https://learn.microsoft.com/power-platform/guidance/coe/after-setup
> 
> Feel free to reach out if you need any assistance. We're happy to help!

Has the issue been addressed in the latest CoE release?

**Comment by mohamrizwa** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to a new issue. We’re happy to help anytime. Thanks again for your engagement!
